05_iigo/index.tex

```diff
@@ -305,35 +305,36 @@ \section{Accountability Cycle}
 
-The result of monitoring is intended to be a trigger for the initiation of power transfer, whereby a negative result indicates that at least one rule was broken by the role monitored and the role should be re-elected. Each turn, in addition to monitoring the actions taken by the role in that IIGO session, the election that role held in the previous IIGO session, if one was held at all, is checked for compliance with the rules of power-transfer.
+The result of monitoring is intended to be a trigger for the initiation of power transfer, whereby a declaration of a negative result indicates that at least one rule was broken by the role monitored and the role should be re-elected. Each turn, in addition to monitoring the actions taken by the role in that IIGO session, the election that role held in the previous IIGO session, if one was held at all, is checked for compliance with the rules of power-transfer.
 
-Within the scope of the coursework, we decided to pursue only \emph{one degree of monitoring}, meaning that the powers relating to the accountability cycle will not be monitored themselves. We assume that agents will act in the interest of all the islands in the archipelago. Hence, while the agents might be inclined to break the rules to benefit in some form, it is assumed that the others will negatively see any non-compliant islands based on the assumption that the proposed IIGO system is in place to maintain the welfare of all the islands.
+Figure~ \ref{fig:cycles_in_IIGO} illustrates the reverse nature of the monitoring and power transfer cycles. This is a choice made by design to spread out the process of pre-mature removal from power as a result of wrongdoing. The goal is to avoid malice. If it is one role's power to monitor, it is the other's to initiate and facilitate power transfer. Hence the latter is given a second opinion for cases where the declaration of wrongdoing is not thruthful.
+
+Within the scope of the coursework, we decided to pursue only \emph{one degree of monitoring}, meaning that the powers relating to the accountability cycle will not be monitored themselves. We assume that agents will act in the interest of all the islands in the archipelago. Hence, while the agents might be inclined to break the rules to benefit in some form, it is assumed that the others will negatively see any non-compliant islands based on the assumption that the proposed IIGO system is in place to maintain the welfare of all the islands. We still have rules governing this process (see Rule~\ref{rule:monitoring_1} and Rule~\ref{rule:monitoring_2}), however these rules are not enforced.
+
+The \emph{one degree of monitoring} is another justification for the reverse nature of the monitoring and power transfer cycles. The addition of a second opionion means that a role does not hold the power to both wrongfully declare wrongdoing and hold an election for the same role. 
```

Comment:
**transfer-of-power** cycles

the addition of a WHAT?
I don't understand your last sentence -- can oyu revise